app/src/main/res/values/attrs.xml

```diff
@@ -1,12 +1,36 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <attr name="header" format="string" />
```

Comment:
these attires values are not part of any style able

Reply:
But we are using it in XML files like app:header="xzz_title", though the setHeader is not being used in the CustomView Classes.